### DISCUSSION TITLE
User profile avatar_url is missing when logged in using Google OAuth provider

### DISCUSSION THREAD

**Discussion by rshtj** [supabase/supabase#2167] [Questions] [answered]
I believe there used to be a property name called avatar_url when accessing supabase.auth.user() to get current logged-in users details when user logs in using Google OAuth provider.

I used this URL to display the user's profile image a couple of weeks back, but I removed the code and didn't have the code to access the image now. But when I try to console log to check the exact property to display the image again, I couldn't find it.

Is there any way to access a user's profile image when logged in using the OAuth provider now?

**Comment by silentworks** [comment]
I've never tested with the Google OAuth provider before but I don't remember seeing this with the GitHub provider.

**Reply by rshtj** [reply]
I am pretty sure it exists in any auth provider, the image can be accessed by avatar_url property. (This GitHub doc)[https://docs.github.com/en/rest/reference/users#get-the-authenticated-user] shows all the fields you get from a response, it has the avatart_url property I am talking about.

The issue is now resolved when I logged in using another google account.
For future references, you can display images by
```
const user = supabase.auth.user();
const avatarURL = user.user_metadata?.avatar_url;
<img src={avatarURL} />
```

**Comment by rshtj** [comment, marked as answer]
I found the property I was looking for, here it is

```
const user = supabase.auth.user();
const avatarURL = user.user_metadata?.avatar_url;
<img src={avatarURL} />
```
If I log in with another google account I can see it, I think if you use the same email id to log in using email and password, and then use the same one to log in using provider, the property won't be there under user_metadata.
